docs/troubleshooting.md

```diff
@@ -124,6 +124,24 @@ With this config you can start your controller as usual. Any workers will need t
 k0s worker --profile coreos [TOKEN]
 ```
 
+### k0s nodes faile to get to 'Ready' State 
+
+This could be a sign that your machine  are the same among your hosts. First run
```

Comment:
```suggestion
This could be a sign that your [machine IDs](https://man7.org/linux/man-pages/man5/machine-id.5.html) are the same among your hosts. First run
```